pkg/virt-handler/node-labeller/node_labeller.go

```diff
@@ -68,22 +69,24 @@ type NodeLabeller struct {
 	clusterConfig           *virtconfig.ClusterConfig
 	hypervFeatures          supportedFeatures
 	hostCapabilities        supportedFeatures
+	supportedMachines       []libvirtxml.CapsGuestMachine
 	queue                   workqueue.TypedRateLimitingInterface[string]
 	supportedFeatures       []string
 	cpuModelVendor          string
 	volumePath              string
 	domCapabilitiesFileName string
 	cpuCounter              *libvirtxml.CapsHostCPUCounter
+	guestCaps               []libvirtxml.CapsGuest
```

Comment:
I'm not sure how large these structs are, and they could grow in the future. If possible, I'd prefer not to keep them in memory for the entire handler lifecycle.